.github/workflows/release.yml

```diff
@@ -25,17 +25,49 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go
+      - name: cache cargo
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            tools/optimizer-server/target/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('tools/optimizer-server/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo
       - name: build nydus-snapshotter
         run: |
           go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.51.2
           export PATH=$PATH:$(go env GOPATH)/bin
           make
+      - name: build optimizer
+        run: |
+          rustup component add rustfmt clippy
+          make optimizer
+          make static-optimizer-server
       - name: upload artifacts
         uses: actions/upload-artifact@v3
         with:
           name: nydus-snapshotter_artifacts
           path: |
             bin/containerd-nydus-grpc
+            bin/02-optimizer-nri-plugin
```

Comment:
Is it any special to add the prefix `02-`?

Reply:
> Is it any special to add the prefix `02-`?

We can not load the NRI plugin without the NRI plugin index `02-`. I don't know if this is a bug, but we can add this prefix when installing `optimizer-nri-plugin`.